### ISSUE TITLE
Password sometimes shows incorrect when correct

### ISSUE THREAD

**Issue by coltonp2022** [datastorm-open/shinymanager#187]
Hello,

I am not sure exactly whether this issue may be something working within my own UI or if it is just a bug within the shinymanager package itself and I am hoping to help sort it out. I am using my own csv database for credentials and they are hashed using the scrypt package; however, even when they were not hashed I was still having this problem. Sometimes when I hit the login button, it says that my credentials are incorrect even when I know they are correct. I have tested it multiple times with different logins and I am not quite sure why it is doing this. 

I am using the secureapp function rather than the auth_ui function, and I am going to try to use the auth_ui function instead to see if that solves the issue tomorrow; however, I am wondering if anyone else running into this issue? 

Thanks for any help with this.

**Comment by coltonp2022** [comment]
A couple more clarification points. The app will say that my credentials are incorrect; however, when I leave them in and then hit the login button again, it will log in just fine. It is almost as if there is some amount of buffering time that shinymanager needs to sort out the password itself. I have used shinyauthr prior with the same app and I did not have the issue in that form; however, I found the customizability of the login page intriguing and went this route.
